tokens/stnear.json

```diff
@@ -1,11 +1,11 @@
 {
     "ethereum_address": "",
-    "aurora_address": "",
+    "aurora_address": "0x07f9f7f963c5cd2bbffd30ccfb964be114332e30",
     "near_address":"meta-pool.near",
     "name": "Staked NEAR",
-    "symbol": "STNEAR",
+    "symbol": "stNEAR",
```

Comment:
this should match the `ft_metadata()` in NEAR smart contract which contains "STNEAR", not "stNEAR"

Reply:
Thanks, updated